Advanced_3/Task/MainWindow.xaml.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Library;
+using Library.Tree;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace Task
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private const string BaseDirectory = "C:\\Users\\Aliaksandr_Karneyeu\\source\\CryptoJack";
```

Comment:
it would be better to avoid hardcoded directory path - it is lead to application crash

Reply:
Fixed